packages/bridge-status-controller/CHANGELOG.md

```diff
@@ -7,8 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [29.2.0]
```

Comment:
Is there a reason why we are bumping the minor here? The minor usually represents new things that developers can use, but it seems like there are just fixes to this package?